presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -19,6 +19,29 @@ Presto Geospatial functions support the Well-Known Text (WKT) form of spatial ob
 * ``MULTIPOLYGON (((0 0, 4 0, 4 4, 0 4, 0 0), (1 1, 2 1, 2 2, 1 2, 1 1)), ((-1 -1, -1 -2, -2 -2, -2 -1, -1 -1)))``
 * ``GEOMETRYCOLLECTION (POINT(2 3), LINESTRING (2 3, 3 4))``
 
+The SphericalGeography type provides native support for spatial features represented on "geographic" coordinates (sometimes called "geodetic" coordinates,
+or "lat/lon", or "lon/lat"). Geographic coordinates are spherical coordinates expressed in angular units (degrees).
+
+The basis for the Geometry type is a plane. The shortest path between two points on the plane is a straight line. That means calculations on geometries (areas, distances, lengths,
+intersections, etc) can be calculated using cartesian mathematics and straight line vectors.
+
+The basis for the SphericalGeography type is a sphere. The shortest path between two points on the sphere is a great circle arc. That means that calculations on geographies
+(areas, distances, lengths, intersections, etc) must be calculated on the sphere, using more complicated mathematics. More accurate measurements that take the actual spheroidal
+shape of the world into account are not supported.
+
+Because the underlying mathematics is much more complicated, there are fewer functions defined for the geography type than for the geometry type. Over time, as new algorithms are
+added, the capabilities of the geography type will expand.
+
+The units returned by the measurement functions (ST_Distance, ST_Length, ST_Area) are in meters.
+
+Use ST_GeometryFromText and ST_GeomFromBinary functions to create geometry objects from WKT or WKB.
```

Comment:
Move this sentence right after `Presto Geospatial functions support the Well-Known Text (WKT) form of spatial objects:` and update that sentence to mention `WKB`.